training/evaluate.py

```diff
@@ -427,6 +427,7 @@ def check_valid_combination(dataset, feature):
         option_manager = BaseOptions(args.model, args.dataset, args.feature)
         option_manager.parse()
         opt = option_manager.option
+        os.makedirs(opt.results_dir, exist_ok=True)
```

Comment:
Is this really necessary?

Reply:
Yes. If we remove this line, we need to make a new directory manually when we evaluate the model using unav100 after we have trained models for clotho-moment.

Originally, train.py includes ```option_manager.clean_and_makedirs()```, but if we try to evaluate the models using test split of clotho-moment, this line deletes the folder including the checkpoints.

Reply:
OK.